instrumentation/opentelemetry-instrumentation-boto3sqs/src/opentelemetry/instrumentation/boto3sqs/__init__.py

```diff
@@ -43,10 +43,18 @@
     unwrap,
 )
 from opentelemetry.propagators.textmap import CarrierT, Getter, Setter
+from opentelemetry.semconv._incubating.attributes.messaging_attributes import (
+    MESSAGING_DESTINATION_NAME,
+    MESSAGING_MESSAGE_CONVERSATION_ID,
+    MESSAGING_MESSAGE_ID,
+    MESSAGING_OPERATION,
```

Comment:
Thanks for contributing!

`MESSAGING_OPERATION` is [also deprecated](https://github.com/open-telemetry/opentelemetry-python/blob/c9ad4bc2cefc64a918f0cda3cf76c64379c87aea/opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/messaging_attributes.py#L186-L189); please could you switch to [MESSAGING_OPERATION_TYPE](https://github.com/open-telemetry/opentelemetry-python/blob/c9ad4bc2cefc64a918f0cda3cf76c64379c87aea/opentelemetry-semantic-conventions/src/opentelemetry/semconv/_incubating/attributes/messaging_attributes.py#L196C1-L200) instead.